### PR TITLE
Add MyAnimeList.Vip presence

### PR DESCRIPTION
**Pull request by vahya**
Screenshot 2021-07-05 175428
Screenshot 2021-07-05 175509
Screenshot 2021-07-05 175643
Screenshot 2021-07-05 175701
Screenshot 2021-07-05 175722

### CODE REVIEW

File: websites/M/MyAnimeList.Vip/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.4",
+  "service": "MyAnimeList.Vip",
+  "author": {
+    "name": "वाह्य",
+    "id": "819696394180100106"
+  },
+  "category": "anime",
+  "logo": "https://i.imgur.com/Refe3ci.png",
```
Comment (ririxidev) [inline websites/M/MyAnimeList.Vip/dist/metadata.json:9]: low quality logo

**Review by rxri — CHANGES_REQUESTED** [review]
support every subpage

**Review by rxri — APPROVED** [review]

File: websites/M/MyAnimeList.Vip/presence.ts [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    startTimestamp: browsingStamp
+  },
+    { pathname } = document.location;
+
+  if (pathname === "/") {
+    if (document.location.search) {
+      presenceData.details = `Procurando por ${document.location.search.substring(
+        3
+      )}`;
+    } else presenceData.details = "Explorando MyAnimeList.vip";
```
Comment (ririxidev) [inline websites/M/MyAnimeList.Vip/presence.ts:31]: wait actually you can make it simplier
```suggestion
      presenceData.details = document.location.search ? `Procurando por ${document.location.search.substring(
        3
      )}` : "Explorando MyAnimeList.vip";
```

**Review by rxri — CHANGES_REQUESTED** [review]
and prettify it yonk xd

File: websites/M/MyAnimeList.Vip/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.4",
+  "service": "MyAnimeList.Vip",
```
Comment (Bas950) [inline websites/M/MyAnimeList.Vip/dist/metadata.json:3]:
```suggestion
  "service": "MyAnimeList.vip",
```
I would do lowercase "v"...

**Review by rxri — DISMISSED** [review]

**Comment by ririxidev** [comment]
aaaa lol, change folder name regarding Bas' review lmaaooo

**Review by rxri — APPROVED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**